assets/js/Line.jsx

```diff
@@ -95,6 +97,26 @@ function Line({
           readOnly={readOnly}
         />
       )}
+      {line === 'Silver' && (
+        <div>
+          <label htmlFor="chelsea_boarding">
+            Boarding Announcements
```

Comment:
Shouldn't this be "Bridge" Announcements? (it looks like there are also some parts of the code that inconsistently use "boarding" vs. "bridge")

Reply:
yeah, will change